core/tests/wdio.conf.js

```diff
@@ -15,9 +15,10 @@ var chromeVersion = (args[0] == 'DEBUG=true') ? args[6] : args[5];
 var chromedriverPath =
 './node_modules/webdriver-manager/selenium/chromedriver_' + chromeVersion;
 
-// To record videos of the failed test suites locally,
-// update the value of LOCAL_VIDEO_RECORDING_IS_ENABLED to 1.
-var LOCAL_VIDEO_RECORDING_IS_ENABLED = 0;
+var spw = null;
```

Comment:
Could you use a variable name that is more meaningful/understandable? I have no idea what this means.

In addition to renaming please add a descriptive comment above this line as well.

Reply:
Done.